### PR TITLE
pythonPackages.flask_migrate: init at 1.7.0

### PR DESCRIPTION
**Pull request by lsix**

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -8774,6 +8774,31 @@ in modules // {
...
+      url = "https://pypi.python.org/packages/source/F/Flask-Migrate/Flask-Migrate-1.7.0.tar.gz";
+      sha256 = "16d7vnaj9xmxvb3qbcmhahm3ldfdhzzi6y221h62x4v1v1jayx7v";
+    };
+
+    # When tests run with python3*, tests should run commands as "python3 <args>",
+    # not "python <args>"
+    patchPhase = ''
+      substituteInPlace tests/test_migrate.py --replace "python" "${python.executable}"
+      substituteInPlace tests/test_multidb_migrate.py --replace "python" "${python.executable}"
+    '';
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: :+1:

**Comment by FRidh** [comment]
Built and tested with all interpreter versions?

**Comment by lancelotsix** [comment]
I have tested `pythonPackages.flask_migrate` and `python3Packages.flask_migrate`. `pypyPackages.flask_migrate` is building and other will follow just after (`pypy3` and `python35`)

**Comment by lancelotsix** [comment]
I have tested with all version except `pypy`. I fail to build `pypy` itself on my machine (and takes ages). I did however cherry-picked my commits on top of the unstable channel (which provides hydra cache for pypy) and it builds / tests fine.

**Comment by FRidh** [comment]
Perfect, thanks!

**Pull request merged**